services/install-bookstack.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+function install {
+  # create service directory
+  mkdir -p /srv/bookstack
+
+  # create yml(s)
+  cat << EOF > /srv/bookstack/bookstack.yml
+version: "2"
+services:
+  bookstack:
+    image: linuxserver/bookstack
+    container_name: bookstack
+    environment:
+      - PUID=\${PUID}
+      - PGID=\${PGID}
+      - DB_HOST=bookstack_db
+      - DB_USER=\${DB_USERNAME}
+      - DB_PASS=\${DB_PASSWORD}
+      - DB_DATABASE=\${DB_DATABASE}
+    volumes:
+      - /srv/bookstack:/root/.bookstack
+    ports:
+      - 6875:80
+    restart: unless-stopped
+    depends_on:
+      - bookstack_db
+  bookstack_db:
+    image: linuxserver/mariadb
+    container_name: bookstack_db
```

Comment:
Delete the container_name field

Reply:
Same here

Reply:
Wonder if you can get it working with these changes?